pandas/core/indexes/datetimelike.py

```diff
@@ -790,6 +806,12 @@ def __sub__(self, other):
             elif is_integer_dtype(other) and self.freq is None:
                 # GH#19123
                 raise NullFrequencyError("Cannot shift with no freq")
+
```

Comment:
in next pass remove the extra line, want to be consistent on formatting

Reply:
Will do.